packages/esm-tb-app/src/views/patient-summary/tb-patient-summary.component.tsx

```diff
@@ -42,47 +43,52 @@ const TBSummaryOverviewList: React.FC<PatientChartProps> = ({ patientUuid }) =>
         key: 'type',
         header: t('type', 'Type'),
         encounterTypes: [config.encounterTypes.tbProgramEnrollment],
-        getObsValue: (encounter) => {
+        getObsValue: ([encounter]) => {
```

Comment:
Was this formatting intentional?

Reply:
Yah, the `getObsValue` function from the `SummaryCard` component expects an array of encounters.